docs/JSON-RPC.md

```diff
@@ -17,6 +17,9 @@ It can be generated like this:
 $ openssl rand -base64 10 > /file/with/a/secret.txt
 ```
 
+The JSON-RPC server defaults to listening on the local loopback network interface (127.0.0.1).  This can be optionally changed by using the `--jsonrpcbindip <ip address>` command line option.
```

Comment:
Please clarify if IPv6 is also available.

Reply:
I personally cannot test IPv6 at this time.  @dtinth might want to comment, but RPC uses QTcpServer, which in turn gets its bind address from QHostAddress( string ) which, according to the docs, can be either IPv4 or 6.  So, in theory, this should work but I do not want to make that representation as I cannot test it.

Reply:
I’m not using IPv6 either, so I cannot say… May suffice to say that only IPv4 is supported for now.

Reply:
pushed updated docs

Reply:
Ok. Tested ipv6 on Windows 11 (binding to ::1) and I could see a connection via ipv6 but not ipv4 as expected. So you can say that ipv6 is also ok.